nerfstudio/configs/sampling_config.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2022 the Regents of the University of California, Nerfstudio Team and contributors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Config used for sampling rays."""
+
+from dataclasses import dataclass
+from typing import Optional
+
+from nerfstudio.configs.base_config import (
+    InstantiateConfig,
+)
+
+
+@dataclass
+class SamplingConfig(InstantiateConfig):
```

Comment:
I would move this inside the pixel_samplers.py so they're in the same file as the configs, /configs is mostly used for larger method config files

Reply:
Actually it doesn't seem like this is used? is it residual from an older version?

Reply:
ya I should just delete that file